ui/src/index.js

```diff
@@ -10,6 +10,7 @@ import AppKeyVis from '@/apps/keyvis';
 import AppHome from '@/apps/home';
 import AppDemo from '@/apps/demo';
 import AppStatement from '@/apps/statement';
+import AppClusterinfo from '@/apps/clusterinfo';
```

Comment:
The naming convension should be `import AppClusterInfo from '@/apps/clusterInfo`.